### DISCUSSION TITLE
Time selector

### DISCUSSION THREAD

**Discussion by mchuc** [primefaces/community#1553] [PrimeNG] [answered]
once, in old androids,  there was such a nice time selection component, not really, I know why it was replaced. roller skates, namely 2 wheels placed in relation to each other, first you could select the hour and then the minutes, but it was so intuitive and fast, especially on mobile phones; now everything is apple-like, but it is not convenient and intuitive. Will there be such a component?

[https://images.app.goo.gl/vGeoKYgbQ5xvae3h6](https://images.app.goo.gl/vGeoKYgbQ5xvae3h6)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We don't have plans to make such a component. You can select time with PrimeNG Calendar component.
